tests/python/gaia-ui-tests/gaiatest/tests/functional/email/manifest.ini

```diff
@@ -4,6 +4,9 @@
 online = true
 smoketest = true
 
+[test_bug_1116566.py]
```

Comment:
We need a more explicit name here, if this test fails again, it would take longer to know what's the issue.